tests/foreman/ui_airgun/test_syncplan.py

```diff
@@ -127,6 +130,52 @@ def test_positive_end_to_end(session):
         assert not session.syncplan.search(plan_name)
 
 
+@tier2
+def test_positive_end_to_end_custom_cron(session):
+    """Perform end to end scenario for sync plan component with custom cron
+
+    :id: 48c88529-6318-47b0-97bc-eb46aae0294a
+
+    :expectedresults: All CRUD actions for component finished successfully
+
+    :CaseLevel: Integration
+    """
+    plan_name = gen_string('alpha')
+    description = gen_string('alpha')
+    cron_expression = gen_choice(valid_cron_expressions())
+    with session:
+        startdate = (
+                session.browser.get_client_datetime() + timedelta(minutes=10))
+        # Create new sync plan and check all values in entity that was created
+        session.syncplan.create({
+            'name': plan_name,
+            'interval': SYNC_INTERVAL['custom'],
+            'description': description,
+            'cron_expression': cron_expression,
+            'date_time.start_date': startdate.strftime("%Y-%m-%d"),
+            'date_time.hours': startdate.strftime('%H'),
+            'date_time.minutes': startdate.strftime('%M'),
+        })
+        assert session.syncplan.search(plan_name)[0]['Name'] == plan_name
+        syncplan_values = session.syncplan.read(plan_name)
+        assert syncplan_values['details']['interval'] == SYNC_INTERVAL['custom']
+        assert syncplan_values['details']['cron_expression'] == cron_expression
+        assert syncplan_values['details']['recurring_logic'].isdigit()
+        time = syncplan_values['details']['date_time'].rpartition(':')[0]
+        assert time == startdate.strftime("%Y/%m/%d %H:%M")
+        # Update sync plan with new description
+        session.syncplan.update(
+            plan_name,
+            {'details.interval': SYNC_INTERVAL['day']
+             }
+        )
+        syncplan_values = session.syncplan.read(plan_name)
+        assert syncplan_values['details']['interval'] == SYNC_INTERVAL['day']
```

Comment:
as you changed sync interval then cron expression will be removed from sync plan, so it will be interesting to verify that it was actually disappeared/is not present in array

Reply:
Yeah, I was also thinking about it. need to know how that I can achieve that as the element will be no longer exist

Reply:
maybe, just `assert not syncplan_values['details']['cron_expression']`. That will check that value is not present anymore in the array. Not the best validation, but at least something

Reply:
done!